plugin-server/src/config/config.ts

```diff
@@ -103,6 +103,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         HISTORICAL_EXPORTS_INITIAL_FETCH_TIME_WINDOW: 10 * 60 * 1000,
         HISTORICAL_EXPORTS_FETCH_WINDOW_MULTIPLIER: 1.5,
         APP_METRICS_GATHERED_FOR_ALL: false,
+        ANONYMOUS_EVENTS_BUFFER_ENABLED_TEAMS: '',
```

Comment:
if you did max team ID, it would be easier to roll-out to everyone over the weekend, we can do max team_id = 2 currently and later change it to 15000 (to exclude the new onboardings)